lizard_connector/connector.py

```diff
@@ -79,9 +80,7 @@ def post(self, url, uuid, data):
                         data to.
             data (dict): Dictionary with the data to post to the api
         """
-        post_url = urllib.parse.urljoin(urllib.parse.urljoin(url, str(uuid)),
-                                        'data')
-        self.perform_request(post_url, data=json.dumps(data))
+        return self.perform_request(url, data=json.dumps(data).encode('utf-8'))
```

Comment:
Is dit de beste plek om te encoden? In principe wil je het het op het laatst mogelijke moment doen, als het eruit gaat.

(idem "gelijk bij binnenkomst")